src/app/util/tipo-servico.ts

```diff
@@ -1,17 +1,17 @@
 import { Servico } from '../interfaces/servico';
 
 export const BANHO:Servico = {
-    id: 1,
+    id: 2,
     nome: 'BANHO'
 }
 
 export const PASSEIO:Servico = {
-    id: 2,
+    id: 12,
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6332425)